capstone/src/main/webapp/lesson.html

```diff
@@ -96,17 +96,17 @@ <h5 class="modal-title" id="ModalCenterTitle">Image</h5>
                     <span aria-hidden="true">&times;</span>
                     </button>
                 </div>
-                <div class="modal-body">
-                        <p> Image Information: </p>
-                        <input type="text" id="image-title" placeholder="Title" required>
-                        <input type="text" id="image-description" placeholder="Description" required>  
-                        <input type="text" id="image-url" placeholder="URL" required>
-                </div>
-                <div class="modal-footer">
-                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                    <button type="button" class="btn btn-primary" onclick="createImage(document.getElementById('image-title').value, document.getElementById('image-description').value, 
-                    document.getElementById('image-url').value)" data-dismiss="modal">Create Image</button>
-                </div>
+                    <div class="modal-body">
+                            <p>Image Information: </p>
+                            <input type="text" id="image-title" placeholder="Title">
+                            <input type="text" id="image-description" placeholder="Description">
+                            <p><input type="text" id="image-url" placeholder="Image Url"></p>
```

Comment:
Paragraphs do not contain inputs. If this was done for visual reasons, use CSS (my guess is you are trying to get `display: block` automatically applied to P tags?)